<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cleanupDialog</class>
 <widget class="QDialog" name="cleanupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File System Analyzer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeWidget" name="treeView">
       <property name="toolTip">
        <string>Selected entries ma be shown in the operating
system or deleted by the context menu.</string>
       </property>
       <column/>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="analyzebutt">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Selects a directory and date and then finds all
files and folders older than the given date. The
files are shown in a tree.</string>
         </property>
         <property name="text">
          <string>Analyze...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Cancel will stop a running analysis. OK will
close the dialog.</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>cleanupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>595</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>cleanupDialog</receiver>
   <slot>doBreak()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>626</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>analyzebutt</sender>
   <signal>clicked()</signal>
   <receiver>cleanupDialog</receiver>
   <slot>doAnalyze()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>452</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>doAnalyze()</slot>
  <slot>doBreak()</slot>
 </slots>
</ui>
